README.md

```diff
@@ -29,7 +29,14 @@ Instantiate an instructor named 'Robin' and call its `givesHomework` method with
 Your Answer:
 
 ```js
-// your code here
+function Instructor(name) {
+  this.name = name;
+  this.givesHomework = function(assignment) {
+    console.log(name + " gives the students " + assignment + " for Friday's homework.")
+  }
+}
+var robin = new Instructor("Robin");
+robin.givesHomework("Intro to Ruby");
```

Comment:
:+1:

Bonus: What would `robin.givesHomework("Hangman")` say, if we ran `robin.name = "Robert";` just before it?